packages/fsapp/src/beta-app/router/history/utils.base.ts

```diff
@@ -66,7 +66,7 @@ const matchPath = (path: string | undefined, route: Route) => {
 
 const buildMatcher = async (
   route: Route,
-  tab: Tab = '',
+  tab: Tab = { id: '' },
```

Comment:
This should probably just be nullable with no default and use optional chaining on line 85.

A `tabAffinity` of `''` doesn't seem particularly useful or semantic.